amoro-web/src/utils/table.ts

```diff
@@ -0,0 +1,26 @@
+/*
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+/**
+ * Calculate table max width
+ */
+export function getTableMaxWidth(columns: Array<{ width: number }>) {
```

Comment:
What's this for?

Reply:
This is a function that scrolls table if it is too long

https://github.com/user-attachments/assets/e64b7868-f08d-4bba-a628-f825760db921